dashboard/src/reducers/config.ts

```diff
@@ -0,0 +1,30 @@
+import { getType } from "typesafe-actions";
+
+import actions from "../actions";
+import { ConfigAction } from "../actions/config";
+import { IConfig } from "../shared/Config";
+
+export interface IConfigState extends IConfig {
+  loaded: boolean;
+}
+
+const initialState: IConfigState = {
+  loaded: false,
+  namespace: "",
+};
+
+const configReducer = (state: IConfigState = initialState, action: ConfigAction): IConfigState => {
+  switch (action.type) {
+    case getType(actions.config.requestConfig):
+      return initialState;
+    case getType(actions.config.receiveConfig):
```

Comment:
Minor comment. Shouldn't this be receivedConfig instead of receiveConfig? Just feels like `receiveConfig` is setting the intention instead of the result.